python/tvm/relax/op/unary.py

```diff
@@ -527,3 +527,23 @@ def isnan(x: Expr) -> Expr:
         The computed result.
     """
     return _ffi_api.isnan(x)  # type: ignore
+
+
+###################### Indicator operators ######################
+
+
+def stop_lift_params(x: Expr) -> Expr:
```

Comment:
I feel it might better for us to categorize this op as a builtin one? It is a nop except the indicator functionality. The “unary”/“binary” class is basically dedicated for arithmetic operations.

Reply:
agree builtin is better